legal-api/src/legal_api/reports/business_document.py

```diff
@@ -470,7 +487,44 @@ def _get_legal_type_description(legal_type: str) -> str:
             }
         },
         'restorationApplication': 'Restoration Application',
-        'restoration': 'Restoration Application',
+        'restoration': { 
+            'fullRestoration': {
+                'CP': FULL_RESTORATION_STR,
+                'BEN': FULL_RESTORATION_STR,
+                'ULC': FULL_RESTORATION_STR,
+                'CC': FULL_RESTORATION_STR,
+                'LLC': FULL_RESTORATION_STR,
+                'SP': FULL_RESTORATION_STR,
+                'GP': FULL_RESTORATION_STR
+            },
+            'limitedRestoration': {
+                'CP': LIMITED_RESTORATION_STR,
+                'BEN': LIMITED_RESTORATION_STR,
+                'ULC': LIMITED_RESTORATION_STR,
+                'CC': LIMITED_RESTORATION_STR,
+                'LLC': LIMITED_RESTORATION_STR,
+                'SP': LIMITED_RESTORATION_STR,
+                'GP': LIMITED_RESTORATION_STR
+            },
+            'limitedRestorationExtension': {
+                'CP': LIMITED_RESTORATION_EXTENSION_STR,
+                'BEN': LIMITED_RESTORATION_EXTENSION_STR,
+                'ULC': LIMITED_RESTORATION_EXTENSION_STR,
+                'CC': LIMITED_RESTORATION_EXTENSION_STR,
+                'LLC': LIMITED_RESTORATION_EXTENSION_STR,
+                'SP': LIMITED_RESTORATION_EXTENSION_STR,
+                'GP': LIMITED_RESTORATION_EXTENSION_STR
+            },
+            'limitedRestorationToFull': {
+                'CP': LIMITED_RESTORATION_TO_FULL_STR,
+                'BEN': LIMITED_RESTORATION_TO_FULL_STR,
+                'ULC': LIMITED_RESTORATION_TO_FULL_STR,
+                'CC': LIMITED_RESTORATION_TO_FULL_STR,
+                'LLC': LIMITED_RESTORATION_TO_FULL_STR,
+                'SP': LIMITED_RESTORATION_TO_FULL_STR,
+                'GP': LIMITED_RESTORATION_TO_FULL_STR
+            }
+        },
         'dissolved': 'Involuntary Dissolution',
         'voluntaryDissolution': 'Voluntary Dissolution',
         'Involuntary Dissolution': 'Involuntary Dissolution',
```

Comment:
I think `dissolution` should look similar to `restoration`.
`
'dissolution': { 
            'voluntary': {},
            'administrative': {},
}
`

Reply:
I don't know what you mean, other than that dissolutions don't use string literals, that I added later after SonarCloud complained. I'm not sure why those other repeated strings didn't create code smells, though...

Reply:
Hey @kzdev420, have you had a chance to read my reply to your suggestion? I'm waiting on you.

Reply:
I think @kzdev420 is referring to how we recently added filing sub-type dissolution support for dissolutions recently.  There might be issues since we introduced that but I think we can leave it as is for now.  If it's an issue we can cut a new ticket for it.